pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -64,8 +66,8 @@ func (pl *InterPodAffinity) EventsToRegister() []framework.ClusterEventWithHint
 		// an unschedulable Pod schedulable.
 		// - Add. An unschedulable Pod may fail due to violating pod-affinity constraints,
 		// adding an assigned Pod may make it schedulable.
-		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.All}},
-		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.UpdateNodeLabel}},
```

Comment:
Should this pending for your PR https://github.com/kubernetes/kubernetes/pull/122292 @sanposhiho

Reply:
Do you wanted to point somewhere else?

Reply:
Yes: https://github.com/kubernetes/kubernetes/pull/122471#discussion_r1438840541